src/Service/IdentificationHelper.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\CaseEntity;
+use App\Entity\Embeddable\Identification;
+use App\Exception\CprException;
+use App\Exception\CvrException;
+use Doctrine\Bundle\DoctrineBundle\EventSubscriber\EventSubscriberInterface;
+use Doctrine\ORM\Event\PreUpdateEventArgs;
+use Doctrine\ORM\Events;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+
+class IdentificationHelper implements EventSubscriberInterface
+{
+    public function __construct(private CprHelper $cprHelper, private CvrHelper $cvrHelper, private PropertyAccessorInterface $propertyAccessor)
+    {
+    }
+
+    /**
+     * @throws CprException
+     * @throws CvrException
+     */
+    public function validateIdentification(CaseEntity $case, string $idProperty, string $addressProperty, string $nameProperty)
+    {
+        /** @var Identification $identification */
+        $identification = $this->propertyAccessor->getValue($case, $idProperty);
+
+        if ('CPR' === $identification->getType()) {
```

Comment:
It may be useful to add

```php
public const IDENTIFIER_TYPE_CPR = 'CPR';
public const IDENTIFIER_TYPE_CVR = 'CVR';
```

to `IdentificationHelper` and use these constants rather than strings throughout the code.

Reply:
Very useful - added `IDENTIFIER_TYPE_CPR ` class constant in df96e8b06fd93385ca8a55bf587036b428e482ad.